runtime/src/utxo.rs

```diff
@@ -56,84 +52,44 @@ pub struct TransactionOutput {
 	/// owner must provide a proof by hashing whole `TransactionOutput` and
 	/// signing it with a corresponding private key.
     pub pubkey: H256,
-
-    /// Unique (potentially random) value used to distinguish this
-	/// particular output from others addressed to the same public
-	/// key with the same value. Prevents potential replay attacks.
-    pub salt: u64,
-}
-
-/// A UTXO can be locked indefinitely or until a certain block height
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
-#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, Hash)]
-pub enum LockStatus<BlockNumber> {
-    Locked,
-    LockedUntil(BlockNumber),
 }
 
 decl_storage! {
     trait Store for Module<T: Trait> as Utxo {
         /// All valid unspent transaction outputs are stored in this map.
         /// Initial set of UTXO is populated from the list stored in genesis.
-        UnspentOutputs build(|config: &GenesisConfig| {
-            config.initial_utxo
+        UtxoStore build(|config: &GenesisConfig| {
+            config.genesis_utxo
                 .iter()
                 .cloned()
                 .map(|u| (BlakeTwo256::hash_of(&u), u))
                 .collect::<Vec<_>>()
         }): map H256 => Option<TransactionOutput>;
 
-
-        /// Total leftover value to be redistributed among authorities.
-        /// It is accumulated during block execution and then drained
-        /// on block finalization.
-        pub LeftoverTotal get(leftover_total): Value;
-
-        /// All UTXO that are locked
-        LockedOutputs: map H256 => Option<LockStatus<T::BlockNumber>>;
+        /// Total reward value to be redistributed among authorities.
+        /// It is accumulated from transactions during block execution
+        /// and then dispersed to validators on block finalization.
+        pub RewardTotal get(reward_total): Value;
     }
 
     add_extra_genesis {
-        config(initial_utxo): Vec<TransactionOutput>;
+        config(genesis_utxo): Vec<TransactionOutput>;
```

Comment:
Since this is a vec, maybe it should be named `genesis_utxos` with a s. That way it's more clear that there acan be multiple of them.

Reply:
ahh yep. will do